airflow/providers/apache/kafka/hooks/kafka_producer.py

```diff
@@ -0,0 +1,95 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from kafka import KafkaProducer
+
+from airflow.hooks.base_hook import BaseHook
+
+
+class KafkaProducerHook(BaseHook):
+    """KafkaProducerHook Class."""
+
+    DEFAULT_HOST = 'localhost'
+    DEFAULT_PORT = 9092
+
+    def __init__(self, conn_id):
+        """
+            Initializes an instance of the Kafka Producer Hook class.
+        :param conn_id
+            The airflow connection ID to use.
+        """
+        super().__init__(None)
+        self.conn_id = conn_id
+        self._conn = None
+        self.server = None
+        self.producer = None
+
+    def get_conn(self) -> KafkaProducer:
+        """
+            Returns a Kafka Producer
+
+        :return:
+            A Kafka Producer object.
+        """
+        if not self._conn:
```

Comment:
`self._conn` is never set. I think you mean

```suggestion
        if not self.producer:
```